cfme/web_ui/menu.py

```diff
@@ -439,16 +439,19 @@ def f(_):
 
         def f2(_):
             args = [toplevel, secondlevel, thirdlevel]
-            loc = cls.ROOT
-            for arg in args:
-                if arg:
-                    if arg.startswith("/"):
-                        loc_to_use = cls.NAMED_LEV_HREF
+            if version.current_version() == 'master' and toplevel == 'Settings':
```

Comment:
sure @psav @jkrocil I'm looking at dashboard and other pages to move this out of menu.py